packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -196,7 +197,23 @@ export const Tooltip = React.forwardRef(
 
     const openTooltip = () => {
       if (tooltipElRef.current && triggerRef.current && !tooltipElRef.current.matches(':popover-open')) {
-        tooltipElRef.current.showPopover()
+        const tooltip = tooltipElRef.current
+        const trigger = triggerRef.current
+        tooltip.setAttribute('popover', 'auto')
```

Comment:
This should always be set so that the browser can correctly expose the right AT nodes. We should not set it as the tooltip is being opened.

Reply:
That makes sense. Although [the tooltip is not exposed to AT](https://github.com/primer/react/pull/3910) when it is not opened but I think it still makes sense to set the initial attributes in the render rather than when it is opened.